### ISSUE TITLE
Product data examples for solar control

### ISSUE THREAD

**Issue by christoph-maurer** [building-envelope-data/api#145]
This issues is based on #138. Please define the product data for solar control similar to
```
              "description": "This resource contains BSDF data about sample VX412005.",
              "hashValue": "d13baa5b91ea95462b1d26b3a3b1874b6be955af5a9630d1d1d0ea9bb981bf0e",
              "locator": "https://oc.ise.fraunhofer.de/ceef5dc1-2501-4ea9-a994-a1ae41ea1f11",
              "format": {
                "id": "a636f9ea-96c3-45ea-8fff-cf7a32978546",
                "timestamp": "2019-09-01T12:00:00+01:00",
                "name": "LBNL/Klems Full",
                "reference": {
                  "publication": {
                    "identifiers": {
                      "doi": "10.1016/j.solener.2017.05.065"
                    }
                  }
                },
                "mediaType": "application/xml; charset=UTF-8"
              },
              "archivedFilesMetaInformation": []
```
Method #144 uses this product data for the demonstration #132.

**Comment by bbuenoun** [comment]
@christoph-maurer by [incidence radiation](https://github.com/ise621/building-envelope-data/blob/2f138b713f67bd38d39d45db0e2b355d5aa35298/schemas/opticalData.json#L343), I wouldn't establish any property as "required", because sometimes we have multiples calls to the schema with the same wavelength (or the same direction), which can be defined in an upper level. Do you agree?

**Comment by bbuenoun** [comment]
@christoph-maurer by [opticalData/dataPoint](https://github.com/ise621/building-envelope-data/blob/2f138b713f67bd38d39d45db0e2b355d5aa35298/schemas/opticalData.json#L80), I wouldn't establish "emergent" as "required", because this is not relevant for hemispherical optical measurements. Do you agree?

**Comment by christoph-maurer** [comment]
> @christoph-maurer by [opticalData/dataPoint](https://github.com/ise621/building-envelope-data/blob/2f138b713f67bd38d39d45db0e2b355d5aa35298/schemas/opticalData.json#L80), I wouldn't establish "emergent" as "required", because this is not relevant for hemispherical optical measurements. Do you agree?

No. For direct-hemispherical measurements,
```
                  "emergence": {
                    "direction": "hemispherical"
                  },
```
is necessary.

**Comment by christoph-maurer** [comment]
> @christoph-maurer by [incidence radiation](https://github.com/ise621/building-envelope-data/blob/2f138b713f67bd38d39d45db0e2b355d5aa35298/schemas/opticalData.json#L343), I wouldn't establish any property as "required", because sometimes we have multiples calls to the schema with the same wavelength (or the same direction), which can be defined in an upper level. Do you agree?

No. To my understanding, there are no multiple "calls". The wavelength is always defined [there](https://github.com/ise621/building-envelope-data/blob/2f138b713f67bd38d39d45db0e2b355d5aa35298/schemas/opticalData.json#L333) and not at upper level.

**Comment by christoph-maurer** [comment]
In the mean-time, several datasets for solar control are available in the product data network.
